libs/initScripts/local.js

```diff
@@ -129,36 +129,38 @@ const run = async () => {
 
       case 'update-delegate-wallet': {
         // Update arbitrary cnode
-        const serviceCount = args[3]
-        if (serviceCount === undefined) throw new Error('update-delegate-wallet requires a service # as the second arg')
+        const serviceNumber = args[3]
+        if (serviceNumber === undefined) throw new Error('update-delegate-wallet requires a service # as the second arg')
         envPath = '../creator-node/compose/env/commonEnv.sh'
-        const account = ethAccounts[parseInt(serviceCount)]
-        await _updateCnodeDelegateWallet(account, envPath, envPath, /* isShell */ true)
+        const account = ethAccounts[parseInt(serviceNumber)]
+        let endpoint = makeCreatorNodeEndpoint(serviceNumber)
+        console.log(endpoint)
+        await _updateConfigFile(account, envPath, envPath, /* isShell */ true, endpoint)
         break
       }
 
       case 'update-cnode-1-delegatewallet':
         // Account 1 - Cnode 1 Delegate Wallet Update
         envPath = '../creator-node/docker-compose/development.env'
-        await _updateCnodeDelegateWallet(ethAccounts[1], envPath, envPath)
+        await _updateConfigFile(ethAccounts[1], envPath, envPath)
```

Comment:
do all the case names also need to be changed from `update-cnode-1-delegatewallet`?

Reply:
these need to be deleted, but i want to handle that on master first - they are no longer relevant with the templated c node format